projects/golang/go/build/conformanceTests/builderBaseRebuildBuildspec.yaml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+env:
+  variables:
+    ARTIFACTS_BUCKET: "eks-d-postsubmit-artifacts"
+    BUILDKIT_PLATFORMS: "linux/amd64"
+    GO_SOURCE_VERSION: "1.16"
+    IMAGE_NAME: "builder-base-development"
+    IMPORT_CACHE_REPOS: ""
+    UPDATE_BASE_IMAGE: "false"
+
+phases:
+  build:
+    commands:
+    - export GOLANG_RPM_DOWNLOAD_TARGETS=$(yq  e "to_entries | .[] | select(.key == \"*GOLANG_VERSION*\") | select(.value != \"$GO_SOURCE_VERSION*\") | .value |= \"download-golang-\" + . | .value" $CODEBUILD_SRC_DIR/builder-base/versions.yaml)
+    - scripts/setup_ecr_credential_helper.sh
+    - mkdir -p builder-base/tmp/golang-downloads
+    - make -C builder-base $GOLANG_RPM_DOWNLOAD_TARGETS
+    - cp ../01/projects/golang/go/$GO_SOURCE_VERSION/rpmbuild/x86_64/golang*.rpm $CODEBUILD_SRC_DIR/builder-base/tmp/golang-downloads
+    - cp ../01/projects/golang/go/$GO_SOURCE_VERSION/rpmbuild/noarch/golang*.rpm $CODEBUILD_SRC_DIR/builder-base/tmp/golang-downloads
+    - make -C builder-base copy-generate-attribution
+    - make -C builder-base images
```

Comment:
doesnt images already run the copy-generate-attribution?

Reply:
no -- `build` and `release` do. I'm running `copy-generate-attribution` and `images` independently in order to skip the checksum generation and validation